src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -2,14 +2,97 @@
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalTimestable.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalTimestable.getTypicalAddressBookReverseClasses;
+import static seedu.address.testutil.TypicalTimestable.getTypicalAddressBookReverseStudents;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
 class SortCommandTest {
 
     @Test
-    void execute() {
-        //Todo discuss how to implement this since it doesnt touch the model.
+    void execute_nameInDescOrder_sortNameAsc() {
+        Model model = new ModelManager(getTypicalAddressBookReverseStudents(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        String sortBy = "name";
+        String directionOfSort = "asc";
+        assertCommandSuccess(new SortCommand(sortBy, directionOfSort), model,
+                String.format(SortCommand.MESSAGE_SUCCESS, "students", sortBy, directionOfSort), expectedModel);
```

Comment:
As stated above, you might want to import the static final variable from `SortCommand.java`! So that the code is more flexible

Reply:
Hmmm, how do i extract it out? I dont think it is feasible because then i would need 4 static variables for the 4 different outputs though

Reply:
Hmm I think 4 static variables are okay. Something like  
```public static final SORT_NAME = "name";
public static final SORT_TIMING = "timing";
public static final SORT_DIRECTION_ASC = "asc";
public static final SORT_DIRECTION_DESC = "desc";
would be good in my opinion so that we won't misspell the words and it'll be easy to change the keywords!

Reply:
Oh u are referring to the "students" and "name" and "asc", i see. Yes that is a good idea, thanks kevin. I thought u were referring to the whole success command. I add the changes now.

Reply:
Ah ok I looked at the above comment and I can see the confusion. Sorry about that!! Thanks Stuart!